skills/README.md

```diff
@@ -0,0 +1,109 @@
+# Skills
+
+## Skills Object
+
+```
+{
+    "name": <name of skill>,
+    "on": <when skill was added>,
+    "by": <who added it>,

```

Comment:
createdBy/ Reviewer 
The purpose here is to ensure that the key signifies the meaning as much as possible

Reply:
https://github.com/Real-Dev-Squad/website-api-contracts/pull/86#discussion_r962937443